src/components/form/HdInputPassword.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div
+    class="passwordInput"
+  >
+    <HdInput
+    :type="currentType"
+    v-model="currentValue"
+    v-bind="$props"
+    :class="{ 'field--hasControl': this.showVisibilityToggle }"
+    />
+
+    <span v-if="showVisibilityToggle"
+      class="field__visibilityToggle"
+      :class="{'field__visibilityToggle--visible': currentType === 'text'}"
+      @click="togglePasswordVisibility"
+    />
+  </div>
+</template>
+
+<script>
+import HdInput from 'hd-blocks/components/form/HdInput.vue';
+
+export default {
+  name: 'HdInputPassword',
+  components: {
+    HdInput,
+  },
+  props: {
+    ...HdInput.props,
+  },
```

Comment:
It is just something that came up to my mind: is it for helping with VSCode validation or so? 
Correct me if I'm wrong but looks like you want to do something like: https://vuejs.org/v2/guide/components-props.html#Disabling-Attribute-Inheritance

Reply:
Actually I'm doing the opposite because here I want to inherit all props from HdInput which is a child in this case.

Reply:
Yep but do you need it? I mean, you're going to forward it all to `<HdInput>`, right?